bin/node/runtime/src/lib.rs

```diff
@@ -540,6 +536,7 @@ impl pallet_staking::Config for Runtime {
 	// Alternatively, use pallet_staking::UseNominatorsMap<Runtime> to just use the nominators map.
 	// Note that the aforementioned does not scale to a very large number of nominators.
 	type SortedListProvider = BagsList;
+	type MaxUnlockingChunks = ConstU32<32>;
```

Comment:
This variable [determines how many unique eras a staker may be unbonding in.](https://github.com/paritytech/substrate/pull/10670/files#diff-cbe1855fd3bef9c6878a793c91771dd0013079a705b0ab1445273ea70dc3b25fR159)
[This is part of unbond logic optimization](https://github.com/paritytech/substrate/pull/10670)

Reply:
Why `32`? What is the ratio between bonding period and this value in Polkadot and Kusama?